blazor/textarea/form-support.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Form Support in Blazor TextArea Component | Syncfusion
+description: Checkout and learn about Form support of the Syncfusion Blazor Textarea component and much more.
+platform: Blazor
+control: Textarea
+documentation: ug
+---
+
+# Form Support in Blazor TextArea component
+
+The TextArea component seamlessly integrates with HTML forms, enabling efficient submission of longer text data. By including TextArea inputs within HTML forms, users can conveniently input multiline text content and submit it as part of form submissions.
+
+This integration enhances the usability of forms, allowing users to provide detailed feedback, enter lengthy descriptions, or input other multiline text data seamlessly.
+
+{% tabs %}
+{% highlight razor %}
+
+@using Syncfusion.Blazor.Inputs
+@using System.ComponentModel.DataAnnotations
+
+<EditForm Model="@formModel" OnValidSubmit="HandleValidSubmit">
+    <SfTextArea @bind-Value="@formModel.FirstName" Placeholder="First Name" FloatLabelType="FloatLabelType.Auto"></SfTextArea>
```

Comment:
Instead of name use "Address" or "Comment" which will be more appropriate for textarea